plugins/source/gcp/resources/services/securitycenter/organization_findings.go

```diff
@@ -0,0 +1,44 @@
+package securitycenter
+
+import (
+	"context"
+
+	pb "cloud.google.com/go/securitycenter/apiv1/securitycenterpb"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+	"github.com/cloudquery/plugins/source/gcp/client"
+)
+
+func OrganizationFindings() *schema.Table {
+	return &schema.Table{
+		Name:        "gcp_securitycenter_organization_findings",
+		Description: `https://cloud.google.com/security-command-center/docs/reference/rest/v1/ListFindingsResponse#ListFindingsResult`,
+		Resolver:    fetchOrganizationFindings,
+		Multiplex:   client.OrgMultiplex,
+		Transform:   transformers.TransformWithStruct(&pb.ListFindingsResponse_ListFindingsResult{}, client.Options()...),
```

Comment:
`ListFindingsResponse_ListFindingsResult` is a `struct` with `Finding`, `StateChange` and `Resource`.
`StateChange` is only returned if we pass `compareDuration` (see docs).
I thought about using the `Finding` struct directly and syncing only `response.Finding`, but it seems `Resource` has useful information.

To have a meaningful primary key I use a path resolver to access the finding name